### PR TITLE
Add slog_term factory

### PR DESCRIPTION
**Pull request by tmccombs**

### CODE REVIEW

**Comment by dpc** [comment]
Oh, nice. I just started the project. :) I'll review tomorrow evening.

File: examples/term_compact.rs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+extern crate slog_config;
+
+use std::{io, fs};
+use std::io::Read;
+use std::path::Path;
+
+use slog::{DrainExt, Logger};
+
+
+fn read_to_string(path: &str) -> io::Result<String> {
```
Comment (dpc) [inline examples/term_compact.rs:12]: I wonder what's the best way to reuse that function. But I guess it does not have to be addressed in this PR.

**Review by dpc — APPROVED** [review]

**Comment by dpc** [comment]
Awesome! Thank you!

File: examples/term_compact.rs (hunk 1 above)
Comment (tmccombs) [reply examples/term_compact.rs:12]: Yeah, I wondered that too.

**Pull request merged**